demo-ui/test/uk/gov/ons/addressIndex/demoui/controllers/SingleMatchTest.scala

```diff
@@ -34,11 +35,12 @@ class SingleMatchTest extends PlaySpec with Results {
     "return an html page" in new WithApplication {
```

Comment:
These tests are a little off, as in, the names do not match the functionality you are testing for. Also take into consideration what your expected is. Remember messages. All I can see are english message tests, not form tests. EG:
"return a form to search" 
`val expected = "<form></form>"`